GitInfo.txt

```diff
@@ -1 +1 @@
-6.0.500-preview.1
+7.0.0-preview.1
```

Comment:
Maybe doesn't need to happen here, but I already put Android as `-preview.6` because we're on .NET 7 Preview 6.